homeassistant/components/light/xiaomi_miio.py

```diff
@@ -210,14 +253,13 @@ def brightness(self):
     @property
     def supported_features(self):
         """Return the supported features."""
-        return SUPPORT_BRIGHTNESS
+        return 0
```

Comment:
This should implement at least `SUPPORT_BRIGHTNESS` since you return the brightness in the property defined above this one?